gems/pending/Gemfile

```diff
@@ -88,6 +89,8 @@ unless ENV['APPLIANCE']
     gem "phantomjs",     "=1.9.8.0",    :require => false
     gem "rspec",         "~>3.5.x",     :require => false
     gem "timecop",       "~>0.7.3",     :require => false
+    gem "vcr",           "~>2.6",       :require => false
```

Comment:
This line is duplicated in the rails app Gemfile:118, please remove the other since this Gemfile is included in the rails app one.